### DISCUSSION TITLE
How many led lights?

### DISCUSSION THREAD

**Discussion by class976** [jomjol/AI-on-the-edge-device#2639] [General]
I have been testing  jomjol with my led light strip that i got from amazon. And  when i try it with 20 leds and above that, i get many crashes  with the camera or freesing with leds on. How many leds can esp32 handle?
Greetings

**Comment by denwald** [comment]
You can calculate the power draw of the LEDs using the very comprehensive [Adafruit guide](https://learn.adafruit.com/adafruit-neopixel-uberguide/powering-neopixels). Maximum draw at full brightness is around 60 mA per LED. Twenty LEDs (20 x 0,06 A = 1,2 A) is most certainly too much for a regular USB charger.

I'm only using three LEDs with good results regarding illumination. I think you should be able to use 5 LEDs without a problem, but I haven't tried it myself.
